src/Mod/Path/PathScripts/PathSelection.py

```diff
@@ -391,6 +391,7 @@ def select(op):
     opsel["MillFace"] = pocketselect
     opsel["Pocket"] = pocketselect
     opsel["Pocket 3D"] = pocketselect
+    opsel["Pocket3D"] = pocketselect
```

Comment:
@sliptonic maybe append `// depricated` to this line so that in future it could be removed?